lib-src/mod-nyq-bench/NyqBench.cpp

```diff
@@ -1721,7 +1721,7 @@ void NyqBench::RecreateToolbar(bool large)
    tb->AddSeparator();
    tb->AddTool(wxID_FIND, _("Find"), mPics[12], _("Find text"));
    tb->AddSeparator();
-   tb->AddTool(ID_MATCH, _("Match"), mPics[13], _("Go to matching paren"));
+   tb->AddTool(ID_MATCH, _("Match"), mPics[13], _("Go to matching parent"));
```

Comment:
This is correct as-is; paren is short for parenthesis.

Reply:
OK. My bad then. I've reverse it. Thanks.